### PR TITLE
Added doc for accessing dn directly from scenario and pipeline

### PR DESCRIPTION
**Pull request by toan-quach**

### CODE REVIEW

File: docs/manuals/core/entities/data-node-mgt.md [hunk 1]
```diff
@@ -62,6 +62,25 @@ The data nodes that are part of a **scenario**, **pipeline** or **task** can be
     task.sales_history
     ```
 
+It is also possible to directly access all data nodes that are part of a **scenario**, **pipeline** as attributes:
```
Comment (FabienLelaquais) [inline docs/manuals/core/entities/data-node-mgt.md]: "that are part of a \*\*scenario\*\*, \*\*pipeline\*\*"
->
"that are part of a \*\*scenario\*\* or a \*\*pipeline\*\*"

File: docs/manuals/core/entities/data-node-mgt.md [hunk 2]
```diff
@@ -62,6 +62,25 @@ The data nodes that are part of a **scenario**, **pipeline** or **task** can be
...
+
+!!! Example
+
+    ```python linenums="1"
```
Comment (FabienLelaquais) [inline docs/manuals/core/entities/data-node-mgt.md]: Please don't add linenums if you don't reference lines of code in your text body.

File: docs/relnotes.md [hunk 3]
```diff
@@ -74,6 +74,7 @@ Published on 2022-06.
      _last_edit_date_ must be used instead.
    - The _edition_in_progress_ attribute of data nodes is now deprecated.<br/>
      _edit_in_progress_ must be used instead.
+   - The data node of a scenario or a pipeline can now be accessed from the scenario and pipeline level.
```
Comment (FabienLelaquais) [inline docs/relnotes.md]: "from the scenario and pipeline level"
->
"at the scenario or pipeline level"

**Review by FabienLelaquais — APPROVED** [review]
Minor changes...

File: docs/manuals/core/entities/data-node-mgt.md [hunk 4]
```diff
@@ -62,6 +62,25 @@ The data nodes that are part of a **scenario**, **pipeline** or **task** can be
     task.sales_history
     ```
```
Comment (jrobinAV) [inline docs/manuals/core/entities/data-node-mgt.md:64]:
```suggestion

# Get all data nodes
```

I would add a section

File: docs/relnotes.md (hunk 3 above)
Comment (jrobinAV) [inline docs/relnotes.md]: It is not in 1.1. And it is not planned to be added to 1.1 for now. It should be in a 1.2 (In progress) section to be added before the 1.1 section!

**Review by jrobinAV — CHANGES_REQUESTED** [review]

**Review by jrobinAV — APPROVED** [review]

**Pull request merged**